libkbfs/mdserver_memory.go

```diff
@@ -336,6 +348,16 @@ func (md *MDServerMemory) Put(ctx context.Context, rmds *RootMetadataSigned,
 		return MDServerError{err}
 	}
 
+	if extra == nil {
```

Comment:
I feel like the caller of MDServer should be doing this, i.e. MDOps. Otherwise, we'd have to duplicate this code among all the MDServer implementations, right?